packages/meditrak-app/app/assessment/Question.js

```diff
@@ -93,7 +93,7 @@ const mergeProps = ({ hasValidationErrorMessage, ...restOfStateProps }, { dispat
         dispatch(validateComponent(screenIndex, componentIndex, validationCriteria, newAnswer));
       }
     },
-    onChangeExtraProps: newProps => dispatch(changeExtraProps(id, newProps)),
```

Comment:
I like this cleanup of `extraProps`. Can we take it one step further and just get rid of this whole thing? It seems like it was only used in the `File` and `Photo` questions

Reply:
Good point, the cleanup wasn't too bad